rest-api-spec/api/nodes.stats.json

```diff
@@ -56,6 +56,10 @@
           "options" : ["node", "indices", "shards"],
           "default" : "node"
         },
+        "_path": {
```

Comment:
No, like pretty and `_source`, we can just support this parameter everywhere, so don't add it to the rest spec.